vnext/Microsoft.ReactNative/IReactPackageBuilder.idl

```diff
@@ -15,4 +15,10 @@ namespace Microsoft.ReactNative {
     void AddModule(String moduleName, ReactModuleProvider moduleProvider);
     void AddViewManager(String viewManagerName, ReactViewManagerProvider viewManagerProvider);
   }
+
+  [webhosthidden]
+  interface IReactPackageBuilderExperimental
```

Comment:
>IReactPackageBuilderExperimental [](start = 12, length = 32)

We must not create new interfaces like this. Why the change is done comparing with the earlier version?